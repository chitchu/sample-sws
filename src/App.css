html, body {
  background-color: #2d2d2d;
}

#root {
  font-family: 'Ubuntu', sans-serif;
  color: #dfece6;
}

#root h1, #root h2, #root h3, #root h4, #root h5, #root strong {
  font-family: 'Ubuntu Condensed', sans-serif;
}

.container {
  width: 1140px;
  max-width: 100%;
  padding-right: 1rem;
  padding-left: 1rem;
  position: relative;
  margin-left: auto;
  margin-right: auto;
}
