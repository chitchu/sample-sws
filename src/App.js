import React from 'react';
import GridContainer from './components/GridContainer';
import NotifyScrollPosition from './components/hoc/NotifyScrollPosition';
const GridWithScrollLoading = NotifyScrollPosition(GridContainer);
// import GridList from './components/GridList';
// import mock from './mock.json';
import './App.css';

const App = () => (
  <div className="container">
    {/*<GridList items={mock.data} />*/}
    <GridWithScrollLoading loadStrategy="default" />
  </div>
);

export default App;
