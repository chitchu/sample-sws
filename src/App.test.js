import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

describe('<App/>', () => {
  it('renders without crashing', () => {
    const result = shallow(<App />);
    expect(toJson(result)).toMatchSnapshot();
  });
});
