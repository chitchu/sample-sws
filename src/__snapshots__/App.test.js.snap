exports[`<App/> renders without crashing 1`] = `
<div
  className="container">
  <NotifyScrollPosition
    loadStrategy="default" />
</div>
`;
